### ISSUE TITLE
Add scenarios

### ISSUE THREAD

**Issue by probonopd** [sticilface/Melvanimate#22]
It would be cool if one could have scenarios, i.e., set multiple ESPs running Melvanimate at once.

**Comment by sticilface** [comment]
could you explain a little bit more about what you mean by scenarios?  and what you would like to achieve?

**Comment by probonopd** [comment]
Presets that span more than one device, e.g., set LED strip on ESP #1 to blue and LED strip on ESP #2 to orange.

**Comment by sticilface** [comment]
I'm going to have to say that would require a crazy amount of work.  I certainly have no idea how the GUI would have to look to implement that in a useable way.  I'd say a much much easier way would be to just use MQTT and set them that way!  no?

**Comment by probonopd** [comment]
MQTT could certainly be used (by more advanced users). But have a look at the Hue apps for "easy" scenarios... ;-)

**Comment by sticilface** [comment]
i was thinking about this and a stand alone app would be the way forward.  i bit like hue works...

**Comment by probonopd** [comment]
Why not: define a scenario on say ESP#1 and then it syncs this to all the other ESPs using the existing we-find-each-other method? Like peer-to-peer filesharing?

**Comment by probonopd** [comment]
Don't know but maybe an existing app like http://www.blynk.cc/ could be used/interfaced with? There is a library for esp8266/Arduino: https://github.com/blynkkk/blynk-library https://blog.adafruit.com/2016/03/08/talk-to-huzzah-from-your-smartphone-using-blynk-cc-hackaday-blynk_app/

**Comment by sticilface** [comment]
my thoughts were that u'd want them to be different.  It would not be that hard to implement it so that the current settings are synced across different devices.  but for some of the effect to work well like that they would all have to be the same configuration... however, i think the power would come from being able to set a scene.. like one blue, one red, one orange... etc... like different bulbs in a hue arrangement... that would be quite hard to implement in one web front end as it is set up at the moment... 

having said that... maybe it is not worth re-inventing the wheel... and this is what philips hue is really good at... again though that would be a lot of work..

**Comment by probonopd** [comment]
Yes, every ESP should be able to run its own settings, so I meant sync the settings for each ESP across the ESP network...
